src/AudioUnit/AudioUnit.cs

```diff
@@ -1954,7 +2011,8 @@ public enum AUSpatializationAlgorithm : uint
 		Hrtf = 2,
 		SoundField = 3,
 		VectorBasedPanning = 4,
-		StereoPassThrough = 5
+		StereoPassThrough = 5,
+		HRTFHQ = 6,
```

Comment:
I think this is an acronym for Head-related-transfer-function and no idea about the HQ.. but will lower everything except the H.